spec/factories/litigations.rb

```diff
@@ -35,10 +35,6 @@
     at_issue { 'At issue Lorem ipsumumum' }
     visibility_status { Litigation::VISIBILITY.first }
 
-    factory :published_litigation do
```

Comment:
There are global `traits` in `factories/visiblity_status.rb` file for that.

Reply:
ahhhh lame me

Reply:
I guess that's not very intuitive, only finding example could tell that there are some globals.